plugins/chat/assets/javascripts/discourse/services/chat-channel-pane-subscriptions-manager.js

```diff
@@ -36,7 +36,7 @@ export default class ChatChannelPaneSubscriptionsManager extends ChatPaneBaseSub
 
   handleThreadOriginalMessageUpdate(data) {
     const message = this.messagesManager.findMessage(data.original_message_id);
-    if (message.thread) {
+    if (message?.thread) {
```

Comment:
technically not relate to the sql request but I saw it could fail while working on it